apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { CoinBalance } from '~/ui/CoinBalance';
+import { Heading } from '~/ui/Heading';
+import { SenderRecipientAddress } from '~/ui/SenderRecipientAddress';
+
+type Recipient = {
+    address: string;
+    coin?: {
```

Comment:
As a follow-up we should generalize this type, but just clocking this for later.